app/helpers/votes_helper.rb

```diff
@@ -20,40 +20,45 @@ def split_hash(key, value, model)
       val = split_presence(value)
     when 'votecode'
       val = split_votecode(value)
-    when 'open'
-      val = split_open(value)
+    when 'status'
+      val = split_status(value)
     when 'title'
       val = split_votecode(value)
+    when 'present_users'
+      val = split_present_users(value)
     else
       val = value.to_s
     end
 
-    unless val.blank?
+    if val.blank?
```

Comment:
Not sure either - it could be breaking it out into a method or something.

But this one looks fine